iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/parser/StatementGenerator.java

```diff
@@ -510,7 +510,7 @@ public static CreateTimeSeriesStatement createStatement(TSCreateTimeseriesReq re
     final long startTime = System.nanoTime();
     // construct create timeseries statement
     CreateTimeSeriesStatement statement = new CreateTimeSeriesStatement();
-    statement.setPath(new PartialPath(req.path));
+    statement.setPath(DEVICE_PATH_CACHE.getPartialPath(req.path));
```

Comment:
The partialPath here should be a timeseries path. I feel it isn’t necessary to cache it.

Reply:
Nice catch! Has reverted